<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QWidget" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_x_min">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>185</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:rgb(145, 145, 145);
}</string>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="display_sum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>33</y>
     <width>181</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #FFE4E1;
    background-color : #C0C0C0;
    text-align: right;
    color: black;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_x_min_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>185</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:rgb(145, 145, 145);
}</string>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_x_min_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>185</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:rgb(145, 145, 145);
}</string>
   </property>
   <property name="text">
    <string>месяцев</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_x_min_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>185</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:rgb(145, 145, 145);
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_result">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color:rgb(255, 161, 116)}
QPushButton:pressed { color: rgb(3, 1, 2)}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_x_min_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>229</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:rgb(145, 145, 145);
}</string>
   </property>
   <property name="text">
    <string>Тип</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_annuitet">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>235</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">Line {
	border-color: rgba(255, 225, 255, 30);
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_result">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>1</y>
     <width>471</width>
     <height>391</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="text_x_min_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>164</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:rgb(145, 145, 145);
}</string>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="display_term">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>93</y>
     <width>181</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #FFE4E1;
    background-color : #C0C0C0;
    text-align: right;
    color: black;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_x_min_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>229</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:rgb(145, 145, 145);
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="display_percent">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>153</y>
     <width>181</width>
     <height>47</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #FFE4E1;
    background-color : #C0C0C0;
    text-align: right;
    color: black;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>text_x_min_2</zorder>
  <zorder>display_term</zorder>
  <zorder>text_x_min_3</zorder>
  <zorder>display_percent</zorder>
  <zorder>display_sum</zorder>
  <zorder>text_x_min</zorder>
  <zorder>button_back</zorder>
  <zorder>text_x_min_4</zorder>
  <zorder>text_x_min_5</zorder>
  <zorder>text_x_min_6</zorder>
  <zorder>button_result</zorder>
  <zorder>text_x_min_7</zorder>
  <zorder>radioButton_annuitet</zorder>
  <zorder>radioButton_diff</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>table_result</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
